### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by while-false**
First take on content for the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+# pyprotoc_plugin
+
+The package provides a small `python3` library as well as some `bazel` templates to
+make it easier for developers using `bazel` to write `protoc` plugins and generate custom code from protobuf files.
+
+## Creating a plugin
+
+A `protoc` plugin is any executable, named `protoc-gen-.*`, that can read and write `protoc` compatible input and output from `stdin`/to `stdout`.
```
Comment (rjhuijsman) [inline README.md:8]: Typo: `/to`?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+```python
+#!/usr/bin/env python3
+from pyprotoc_plugin.plugins import ProtocPlugin
+from google.protobuf.descriptor_pb2 import FileDescriptorProto
+
+class NopPlugin(ProtocPlugin):
+
+    def process_file(self, proto_file: FileDescriptorProto) -> None:
+        pass
```
Comment (rjhuijsman) [inline README.md:20]: Maybe explicitly call out in a comment that this example explicitly does nothing.

**Review by rjhuijsman — APPROVED** [review]
Very nice!

To consider for future expansion: I'd love if we had an `examples` directory in this repo with at least one end-to-end example that puts all the pieces you describe here together.

**Comment by while-false** [comment]
> To consider for future expansion: I'd love if we had an `examples` directory in this repo with at least one end-to-end example that puts all the pieces you describe here together.

I think we already have a Todo item/issue for that. Feel free to take it on if you'd like! :wink:

File: README.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+# Use the generated files to build a cc_library
+cc_library(
+    name="object_store_eventuals",
+    srcs=[":object_store_eventuals_generated"],
+    deps=[
+        ":object_store_grpc",
+        "@com_github_3rdparty_eventuals_grpc//:grpc",
+    ],
+    visibility = ["//visibility:public"],
+)
```
Comment (benh) [inline README.md:176]: Oops! Is this supposed to be something else?

**Review by sp33drac3r — COMMENTED** [review]
LGTM

**Pull request merged**